src/components/input/inputText/inputText.test.tsx

```diff
@@ -24,4 +25,26 @@ describe('<InputText /> component', () => {
         render(createTestComponent({ inputClassName }));
         expect(screen.getByRole('textbox').className).toContain(inputClassName);
     });
+
+    it('renders with a left addon when addonPos is left and addon is a valid string', () => {
+        const addon = 'Left Addon';
+        render(createTestComponent({ addon, addonPos: 'left' }));
+        expect(screen.getByText(addon)).toBeInTheDocument();
+    });
+
+    it('renders with a right addon when addonPos is right and addon is a valid string', () => {
+        const addon = 'Right Addon';
+        render(createTestComponent({ addon, addonPos: 'right' }));
+        expect(screen.getByText(addon)).toBeInTheDocument();
+    });
```

Comment:
If we use the `order` css property as suggested below, we can just write one test to check that the addon is rendered and we don't need to check for class names